week2/exercise P1008 in LuoGu.cpp

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+int main()
+{
+	int num1, num2, num3;

```

Comment:
`num1`, `num2`, `num3` as well as `i`, `j`, `k` are not good names. Reviewers has to go through the implementation to get the idea what they refer to. `num`, `double`, `triple` will be better.